release-schema.json

```
@@ -4,7 +4,27 @@
       "properties": {
         "coveredBy": {
           "title": "Covered by",
-          "description": "The treaties that the contracting process is covered by.",
+          "description": "The treaties and directives that the contracting process is covered by.",
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": "string",
+            "minLength": 1
+          },
+          "codelist": "coveredBy.csv",
+          "openCodelist": true,
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      }
+    },
+    "Lot": {
+      "properties": {
+        "coveredBy": {
+          "title": "Covered by",
+          "description": "The treaties and directives that the contracting process for this lot is covered by.",
```

Slight amendment to clarify that the field should only be used for the treaties and directives for the specific lot.

```suggestion
          "description": "The treaties and directives that the lot is covered by.",
```